.env

```diff
@@ -0,0 +1,2 @@
+DB_HOST=mongodb+srv://stanislav_pravdiuk:Sp23038888@db.byyndvr.mongodb.net/phonebook?retryWrites=true&w=majority
```

Comment:
.env file shouldn't be visible on github its data must be confidential. Add this file to gitignore